ckan/config/config_declaration.yaml

```
@@ -1193,6 +1193,13 @@ groups:
       - key: ckan.site_custom_css
         description: Custom CSS directives to include on all CKAN pages.
         editable: true
+      
+      - key: ckan.default_collapse_facets
+        type: bool
+        example: "false"
```

Let's show the default value for clarity

```suggestion
        default: False
```